cepheus/pom.xml

```diff
@@ -324,6 +326,29 @@
                     <path>/${tomcat.app}</path>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <doCheck>false</doCheck>
+                    <doUpdate>false</doUpdate>
+                    <revisionOnScmFailure></revisionOnScmFailure>
+                    <items>
+                        <item>buildNumber\d*</item>
+                        <item>timestamp</item>
+                    </items>
+                    <format>#{0} ({1,date,yyyy-MM-dd HH:mm:ss})</format>
+                </configuration>
+            </plugin>
```

Comment:
have you tried this, is that actually generating build numbers? I believe it needs a scm definition.